agreements/agreements-service.yaml

```diff
@@ -69,27 +64,21 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/Errors'
-
-  /agreements/{ca-number}:
+                $ref: 'https://raw.githubusercontent.com/Crown-Commercial-Service/ccs-scale-api-definitions/BAT_Updates_2/common/CCS_Standard_API_Schema.yaml#/components/schemas/Errors'
+#
+  /agreements/{agreementID}:
     get:
       summary: Returns details of the specified Commercial Agreement
       description: Returns the definition of the specified Commercial Agreement
       tags:
-      - get-agreement-detail
+      - Agreement
       operationId: get-agreement-detail
       parameters:
-        - name: ca-number
-          in: path
-          description: Commercial Agreement unique 'number'
-          example: RM3733
-          required: true
-          schema:
-            type: string
-        - $ref: '#/components/parameters/x-correlation-id'
-        - $ref: '#/components/parameters/x-causation-id'
-        - $ref: '#/components/parameters/x-channel'
-        - $ref: '#/components/parameters/x-client'
+        - $ref: 'https://raw.githubusercontent.com/Crown-Commercial-Service/ccs-scale-api-definitions/BAT_Updates_2/common/CCS_Standard_API_Params.yaml#/components/parameters/AgreementParam'
```

Comment:
Just curious - any reason why `agreementID` is referred to as `AgreementParam` (and there are others like this). Not that important - just not particularly clear when reading this document what `AgreementParam` refers to (without following the link and finding the details).

Also - is there a bit of a mix of styles for the params - `AgreementParam` and `x-correlation-id` for example? (I know they are path params and headers - just wonder if they should be more consistent)?

Not that important and not precious about it - just raising as an observation 😄

Reply:
Thanks - good point. Will need to discuss with the architects as these came from different people (Nanu for the x-* parameters, Nick for the Params).  The x-* params are generally headers and the others are path parameters and it seems OK to use different conventions for the types as headers are usually x-*. Open for opinions of course.